### ISSUE TITLE
Add activity indicators 

### ISSUE THREAD

**Issue by socketpair** [s-macke/jor1k#113]
I mean something like attached example
![preloader_flash_1](https://cloud.githubusercontent.com/assets/1157412/12807618/3a718828-cb32-11e5-89b3-f7d2d0e26e70.gif)

- [ ] CPU
- [ ] networking
- [ ] disk

And if you have a time :) also that:
- [ ] sound
- [ ] and maybe event terminal
- [ ] FPS

**Comment by s-macke** [comment]
Yes, such status activity indicators would be awesome. And a list of files which are retrieved from the server. This would also require a redesign of the website.

This request is a duplicate of https://github.com/s-macke/jor1k/issues/24
